polling_stations/templates/base.html

```diff
@@ -12,6 +12,19 @@
     <p>This is a staging site.</p>
   </div>
 </div>
+{% else %}
+<div class="container">
+  <div class="card">
+    <h2>Coronavirus</h2>
+    <p><strong>May 2020 scheduled elections</strong>: The Government have announced
+      they plan to postpone the election in May until 2021. We will update
+      ths information on this site once the postponement has become law.
```

Comment:
Typo: ths -> this. I'd probably drop 'on this site' too.

Reply:
cheers. I think it was actually supposed to be "the information on this site", but I've gone with "this information"

Need to make the same fix on WCIVF :D